Formula/bento4.rb

```diff
@@ -0,0 +1,21 @@
+class Bento4 < Formula
+  desc "Full-featured MP4 format and MPEG DASH library and tools"
+  homepage "https://www.bento4.com/"
+  url "https://github.com/axiomatic-systems/Bento4/archive/v1.5.0-615.tar.gz"
+  version "1.5.0-615"
+  sha256 "109d48b75e7ba34d5a0cb98346c098d9e0d29d58b6bc27b90014dbc558a491a4"
+
+  conflicts_with "gpac", :because => "both install `mp42ts` binaries"
+
+  def install
+    cd "Build/Targets/any-gnu-gcc" do
+      system "make", "AP4_BUILD_CONFIG=Release"
+      bin.install Dir["Release/*"].select { |f| File.executable?(f) }
+    end
+  end
+
+  test do
+    system "#{bin}/mp4mux", "--track", test_fixtures("test.m4a"), "out.mp4"
+    assert_predicate File.exist? "out.mp4"
```

Comment:
Ok, done, thanks. (Should this all be part of the Formula Cookbook? It's nicely described how to get to your first PR but once you have to do a change to your formula it's easy to 1) not know what to do, 2) screw up and 3) not remember what you did.)